sqlitecluster/SQLiteNode.cpp

```diff
@@ -1,6 +1,3 @@
-/// /svn/src/sqlitecluster/SQLiteNode.cpp
-/// =======================
-#include <libstuff/libstuff.h>
```

Comment:
We've been down this path a few times -- can we just include `libstuff.h` from the start of every `.cpp` file?  Otherwise we start creating a bunch of header dependencies that are a pain to sort out.  (Some files include it and others don't, so when you remove one header file suddenly you get a bunch of compile time errors that have nothing to do with the header file you removed, etc.)  For simplicity, I vote we continue to include `libstuff.h` at the start of every .cpp file.  If nothing else, it has nothing to do with the current change so I'd welcome it not being done here.